app/views/cart/show.html.erb

```diff
@@ -31,8 +31,8 @@
       <td>
         <center>
           <p><%=quantity%></p>
-		<p><%= button_to "+1", "/cart/#{item.id}/increment",method: :patch %></p> 
-		<p><%= button_to "-1", "/cart/#{item.id}/decrement",method: :patch %></p> 
+		<p><%= button_to "+1", "/cart/#{item.id}/increment",method: :patch %></p>
```

Comment:
I'm pretty sure these can become something like `cart_increment_path` and `cart_decrement_path` but we can take a look at it.